### ISSUE TITLE
WHATWG Working Mode not being followed?

### ISSUE THREAD

**Issue by gsnedders** [whatwg/compat#201]
Despite `PULL_REQUEST_TEMPLATE.md` being in the repo since 1 Apr 2020, searching PRs for a phrase in it shows https://github.com/whatwg/compat/pulls?q=%22are+written+and+can+be%22, which contains a single PR, while there have been 39 PRs in that time period.

This makes it hard to check whether implementation bugs have been written, what tests were authored alongside the change, etc.

**Comment by karlcow** [comment]
Indeed. so [39 PRs since April 2020](https://github.com/whatwg/compat/pulls?q=is%3Apr+created%3A%3E2020-04-01)

On the 39 PRs

* [6 Editorial ](https://github.com/whatwg/compat/pulls?q=is%3Apr+created%3A%3E2020-04-01+Editorial+in%3Atitle)
* [5 Review Draft](https://github.com/whatwg/compat/pulls?q=is%3Apr+created%3A%3E2020-04-01+Review+in%3Atitle+)


I can see only 

* #196
* #185 

All the rest is editorial in nature or about the user agent strings section of the compat spec. 
So that's only 2. The #196 was a modification of something which was already in the compat spec but just simplified, and you raised the issue about missing tests. Thanks!
And for #185, not sure there's a test for this. Editorial nature?

**Comment by karlcow** [comment]
@gsnedders apart of us being more careful when merging, would you suggest something that could help us solve it?

**Comment by gsnedders** [comment]
I think #155 could also be tested, as with it actually being defined what it means to be an alias we can actually test _how_ they're aliased. But in general:

1. People actually using the PR template (honestly just being explicit in the PR that there is no normative change is a win!),
2. The editors ensuring normative changes having tests prior to merging them.

Also the whole UA string section is kinda unclear as to whether it should count as an addition per the working mode. Is it an informative section? Does it have any sort of normative status? Because it seems weird to have an entirely informative top-level section that… sorta defines things?

**Comment by karlcow** [comment]
@gsnedders so for #155 and #196 
Probably  https://github.com/web-platform-tests/wpt/issues/34493 would solve the issue? 

As for UA string section, it is documenting more than being prescriptive. The initial goal, when @miketaylr and me discussed it, was to have it in a separate repo, but Anne recommended we reuse the compat spec instead of creating a new thing. That's for the history bit. :)

**Comment by miketaylr** [comment]
Thanks, we can be more thoughtful about writing tests and filing bugs going forward. The UA string section is a work in progress. We can add tests to it in the future, but right now it's just descriptive. If you feel better about some kind of note inline that says as much, I'm happy to review a PR. :)

**Comment by karlcow** [comment]
1. I'm working on fixing the tests in https://github.com/web-platform-tests/wpt/issues/34493  any contributions, ideas are welcome.
2. About workflow, there's nothing actionable on this issue here, apart let's make it better when we are reviewing. 

Let's close.
